client/src/app-storage.ts

```diff
@@ -23,15 +21,6 @@ export class AppStorage {
     constructor(private storage: Storage) {
     }
 
-    public getMaster(): string | undefined {
-        const value = this.storage.getItem(MASTER_KEY)
-        return value ? value : undefined
-    }
-
-    public setMaster(master: string): void {
-        this.storage.setItem(MASTER_KEY, master)
-    }
-
     public getPhysicsFeature(feature: PhysicsFeature): number {
```

Comment:
Yer gonna have to make these storage methods async if you wanna load them from the server. Will that be easy right now?

Reply:
Done that a few times.  Not hard at all.  I use lots of asyc now even including loading the WASM module.